### ISSUE TITLE
schemas fail swagger-cli validation

### ISSUE THREAD

**Issue by rogpeppe** [influxdata/openapi#239]
commit 12c8165b70cb379486592ffbf43ac5a3dde04904

By way of sanity checking, I thought I'd run the schemas through the `swagger-cli` validator, and they don't seem to pass validation:

```
% swagger-cli validate oss.yml
Swagger schema validation failed. 
  Data does not match any schemas from 'oneOf' at #/components/securitySchemes/TokenAuthentication
    Missing required property: $ref at #/components/securitySchemes/TokenAuthentication
    Data does not match any schemas from 'oneOf' at #/components/securitySchemes/TokenAuthentication
      Missing required property: in at #/
      Data does not match any schemas from 'oneOf' at #/
        No enum match for: token at #/scheme
        Data matches schema from 'not' at #/
      Missing required property: flows at #/
      Missing required property: openIdConnectUrl at #/
 
JSON_OBJECT_VALIDATION_FAILED
```

It seems that the security schemas don't match the [OpenAPI 3.1 specification](https://github.com/OAI/OpenAPI-Specification/blob/1fb4f83882e135899fee4c521cc1c6161bb85125/schemas/v3.1/schema.yaml) (for example the `TokenAuthentation` scheme should probably be defined as [`scheme: bearer`](https://github.com/OAI/OpenAPI-Specification/blob/1fb4f83882e135899fee4c521cc1c6161bb85125/schemas/v3.1/schema.yaml#L796-L810) as a `token` scheme doesn't seem to exist.

**Comment by glinton** [comment]
Good catch, we should change that as well as add an additional ci step that runs the validation. Only caveat is that changing "token" to "bearer" shouldn't affect what the header value is set to (the api accepts `Authorization: Token abc123`, but not (maybe not always) `Authorization: bearer abc123`).
